.husky/pre-push

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
Cool!!!

Perhaps we can also add running the linter here to check if everything is ok before pushing and avoid the github action to fail due to linter. What do you think?

Reply:
Sure. In this case, I'd run `lint:fix`.

Reply:
I've noticed `pre-commit` already executes `npm run lint`. Would it be necessary to do it on this hook?

Reply:
No. We need to decide if we want to run it when committing or when pushing. What do you prefer?